src/main/java/org/identityconnectors/ldap/LdapUtil.java

```diff
@@ -154,17 +155,17 @@ public static void addStringAttrValues(Attributes ldapAttrs, String ldapAttrName
      * forward slash (/)
      */
     public static String escapeDNValueOfJNDIReservedChars(String DN) {
-	StringBuilder toBuilder = new StringBuilder();
-	for (int i = 0; i < DN.length(); i++) {
-	    char ch = DN.charAt(i);
-	    switch (ch) {
-	    case '/':
-		toBuilder.append("\\2f");
-		break;
-	    default:
-		toBuilder.append(ch);
-	    }
-	}
+    StringBuilder toBuilder = new StringBuilder();
+    for (int i = 0; i < DN.length(); i++) {
+        char ch = DN.charAt(i);
+        switch (ch) {
+        case '/':
+        toBuilder.append("\\2f");
```

Comment:
This is incorrect indentation (probably caused by automatic conversion of tabs to spaces).

Reply:
Yes, you are right. It was caused by automatic conversion of tabs to spaces. It was fixed in the latest commit.